dbms/tests/integration/test_reload_max_table_size_to_drop/test.py

```diff
@@ -0,0 +1,49 @@
+import time
+import pytest
+import os
+
+from helpers.cluster import ClickHouseCluster
+
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance('node', config_dir="configs")
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+CONFIG_PATH = os.path.join(SCRIPT_DIR, './_instances/node/configs/config.xml')
+
+
+@pytest.fixture(scope="module")
+def start_cluster():
+    try:
+        cluster.start()
+        node.query("CREATE TABLE test(date Date, id UInt32) ENGINE = MergeTree(date, id, 8192)")
```

Comment:
It's better to not use deprecated method for creating MergeTree tables (see https://clickhouse.yandex/docs/en/operations/table_engines/mergetree/).  
```suggestion
        node.query("CREATE TABLE test(date Date, id UInt32) ENGINE = MergeTree() PARTITION BY date ORDER BY id")
```